packages/client-sdk-nodejs/src/config/retry/topic-subscription-retry-strategy.ts

```diff
@@ -5,6 +5,7 @@ import {
   SubscriptionRetryStrategy,
 } from '@gomomento/sdk-core';
 import {TopicSubscriptionRetryEligibilityStrategy} from './topic-subscription-retry-eligibility-strategy';
+import {DEFAULT_SUBSCRIPTION_RETRY_DELAY_MILLIS} from '@gomomento/sdk-core/dist/src/config/retry/subscription-retry-strategy';
```

Comment:
Should also export the name in the `index.ts` of core to avoid the source-based lookup.